assets/sass/modules/_tables.scss

```diff
@@ -14,7 +14,7 @@ table {
 	thead {
 		display: none;
 
-		@include media($wp-admin-bar) {
+		@include breakpoint($wp-admin-bar) {
```

Comment:
Does this work? Harmonium doesn't have a `$sup-admim-bar` breakpoint unless you added one.